python/ray/_private/runtime_env/packaging.py

```diff
@@ -39,8 +39,18 @@ def __new__(cls, value, doc=None):
         return self
 
     GCS = "gcs", "For packages dynamically uploaded and managed by the GCS."
-    S3 = "s3", "Remote s3 path, assumes everything packed in one zip file."
     CONDA = "conda", "For conda environments installed locally on each node."
+    HTTPS = "https", ("Remote https path, "
+                      "assumes everything packed in one zip file.")
```

Comment:
should this also support `HTTP` btw?

Reply:
It could. We would have to add an HTTP option to the Protocol enum and then add supporting logic to some of the other ``packaging.py`` files. I only added HTTPS because that's what the major Git providers use, and partially because of that, it's easier to test. Should I add support for HTTP as well?

Reply:
let's leave just HTTPS for now and we can extend in the future if necessary